### ISSUE TITLE
'Price' object has no attribute 'as_tuple' in update or add product

### ISSUE THREAD

**Issue by anshul315** [mirumee/saleor#430]
I get this error when I try to add or update a product.

<img width="521" alt="screen shot 2016-01-01 at 11 42 01 pm" src="https://cloud.githubusercontent.com/assets/5572005/12071730/51fe2ab6-b0e1-11e5-8f85-95130a21e17e.png">

Not sure if something with my install or code. Please help :)

Have a great new year

**Comment by bogdal** [comment]
Hi @anshul315. It looks like you have an old version of `django-prices` package. Please install the latest version and try again.
```
pip install -U django-prices
```
Best wishes for a Happy New Year! :v:

**Comment by anshul315** [comment]
@bogdal Thanks for the prompt response :).
I tried that but I am still getting the same error so I did a fresh clone and installed all the requirements in a virtual environment. I still get the same error.
My requirements.txt and pip freeze match.

<img width="243" alt="screen shot 2016-01-02 at 9 01 25 am" src="https://cloud.githubusercontent.com/assets/5572005/12073096/74ad99ae-b12f-11e5-87fe-ed9bf0b366ab.png">

I have django-prices==0.4.8
